src/objects/GS_Admin_Checklist_Section__mdt.object

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomObject xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>NPSP Getting Started Page sections</description>
+    <fields>
+        <fullName>Description_Label__c</fullName>
+        <description>The custom label api name used in the description on a section on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>The custom label api name for the Description Label</inlineHelpText>
+        <label>Description Label</label>
+        <length>255</length>
+        <required>true</required>
+        <type>Text</type>
+        <unique>false</unique>
+    </fields>
+    <fields>
+        <fullName>Position__c</fullName>
+        <description>The position in the section list on the Getting Started page</description>
+        <externalId>false</externalId>
+        <fieldManageability>DeveloperControlled</fieldManageability>
+        <inlineHelpText>Is the position of the section</inlineHelpText>
+        <label>Position</label>
+        <precision>2</precision>
+        <required>true</required>
+        <scale>0</scale>
+        <type>Number</type>
+        <unique>true</unique>
+    </fields>
+    <fields>
+        <fullName>Title_Label__c</fullName>
+        <description>The custom label api name used in the title on a section on Getting Started page</description>
```

Comment:
The custom label api name used in the title of a section on the Getting Started page.